paddle/fluid/platform/mkldnn_utils.h

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
There is `paddle/fluid/platform/mkldnn_reuse.h`, why not combine `mkldnn_utils.h` into `mkldnn_reuse.h`?

Reply:
Agreed.

Reply:
@luotao1 , @tensor-tang  mkldnn_utils.h contains routines that are used within tensor.h. We cannot move mkldnn_utils.h into mkldnn_reuse.h or mkldnn_helper.h , as both of those headers are including tensor.h (via operator.h). So we would have tensor.h including mkldnn_reuse.h , mkldnn_reuse.h including tensor.h  ... which is cyclic dependency. That is why it is separate file